UK_Power_Generation_Units_BMRS/Python/Code/ukpowergeneration/ScriptMapQuery/README-Mapping-Match-JA-#.md

```diff
@@ -0,0 +1,98 @@
+# README for Mapping-Match-JA-#.py ("#" = version)
+## Description
+This document is a README for:
+ > Mapping-Match-JA-#.py
+
+Three csv files are required. These are not version controlled, but can be found at: 
+> \Dropbox (Cambridge CARES)\CoMo shared\ja685\BMRS\Script-BMRS-API
+Their names are: 
+> Input-Template.csv
+> generatortripple.csv
+> powerplanttripple.csv
```

Comment:
Please update to:
`dropbox\CoMo shared\ja685\Modelling-Resources\BMRS\Script-BMRS-API`

Reply:
Done.